src/Tools/dotnet-trace/CommandLine/Commands/StreamCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.Diagnostics.Tools.RuntimeClient.Eventing;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal static class StreamCommandHandler
+    {
+        public static async Task<int> Stream(IConsole console, int pid, string output, uint buffersize, string providers)
+        {
+            try
+            {
+                var configuration = new SessionConfiguration(
+                    circularBufferSizeMB: buffersize,
+                    multiFileSec: 0,
+                    outputPath: output,
+                    ToProviders(providers));
+                var binaryReader = EventPipeClient.StreamTracingToFile(pid, configuration, out var sessionId);
+                Console.Out.WriteLine($"SessionId=0x{sessionId:X16}");
+
+                if (sessionId != 0)
+                {
+                    var filePath = $"dotnetcore-eventpipe-{pid}-0x{sessionId:X16}.netperf";
+                    filePath = Path.Combine(@"S:\github.com\jorive\diagnostics\src\Tools\dotnet-trace", filePath);
```

Comment:
Mmm... I think this shouldn't have gotten checked in? Partially my bad since I didn't spot this before checking it out on my local machine just now :-)

Reply:
ooops :(